### ISSUE TITLE
Scrolling content with iframe that has focusable content should not fail `scrollable-region-focusable`

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#4206]
This was reported to us and after testing confirmed that a parent scrolling region which contains an iframe that has focusable content allows you to scroll the parent scrolling region. So this should not fail `scrollable-region-focusable`.

```html
<div id="foo" style="margin-top: 50vh; height: 50vh; overflow-y: scroll;">
  <p style="margin-top: 100vh">Hello World</p>
  <iframe srcdoc="<button>click me</button"></iframe>
  <p style="margin-top: 100vh">Goodbye World</p>
</div>
```

**Comment by dbjorge** [comment]
Verified the example on stable chrome, firefox, safari
